platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -253,4 +257,4 @@ define(
         return MCTSplitPane;
 
     }
-);
+);
```

Comment:
Minor code style issue here, also some unnecessary whitespace. These are causing a [build failure](https://circleci.com/gh/nasa/openmct/4019) on `gulp checkstyle`. Should be able to fix with a single run of `gulp fixstyle`